demo/ts/components/victory-zoom-container-demo.tsx

```diff
@@ -20,6 +19,8 @@ import {
   VictoryTheme
 } from "@packages/victory-core";
 
+type DomainType = {x: number, y: number}
```

Comment:
So it looks like the data that are passed in (allData) are actually a list of x and y coords with a single value rather than a tuple

Reply:
ended up using `CoordinatesPropType` that's already typed out in core